pkg/asset/cluster/tfvars/tfvars.go

```diff
@@ -272,6 +272,13 @@ func (t *TerraformVariables) Generate(parents asset.Parents) error {
 		if err != nil {
 			return err
 		}
+		// Based on the number of workers, we could have the following outcomes:
+		// 1. workers > 0, masters not schedulable, valid cluster
+		// 2. workers = 0, masters schedulable, valid compact cluster but currently unsupported on GCP
+		// 3. workers = 0, masters not schedulable, invalid cluster
+		if len(workers) == 0 {
+			return errors.Errorf("compact clusters with 0 workers are not supported at this time")
+                }
```

Comment:
Weird indent.
```suggestion
        }
```

Reply:
Thanks! I couldn't detect it but fixed it with auto indent.